### PR TITLE
HTML: test ::before and ::after on fieldset

### PR DESCRIPTION
**Pull request by zcorpan**
See https://github.com/whatwg/html/pull/3934

### CODE REVIEW

File: html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/fieldset-generated-content.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<title>fieldset generated content</title>
```
Comment (emilio) [inline html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/fieldset-generated-content.html]: Maybe link to bugs / spec PR / add your name as the author?

**Review by emilio — APPROVED** [review]

File: html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/fieldset-generated-content.html (hunk 1 above)
Comment (zcorpan) [reply html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/fieldset-generated-content.html]: Naw, that is in the git log, and spec link is implied by the dir structure.

File: html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/fieldset-generated-content.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+<fieldset id=test><legend>A</legend>Y</fieldset>
+<fieldset id=ref><legend>A</legend>XYX</fieldset>
+<script>
+test(() => {
+  const testElm = document.querySelector('#test');
+  const refElm = document.querySelector('#ref');
+  assert_equals(testElm.clientWidth, refElm.clientWidth, 'clientWidth');
+  assert_equals(testElm.clientHeight, refElm.clientHeight, 'clientHeight');
+});
+</script>
```
Comment (mstensho) [inline html/rendering/non-replaced-elements/the-fieldset-and-legend-elements/fieldset-generated-content.html]: Missing newline

**Review by mstensho — APPROVED** [review]
Nice one!

**Pull request merged**